stdlib/3/configparser.pyi

```diff
@@ -124,24 +125,23 @@ class RawConfigParser(_parser):
     @overload
     def items(self, section: str, raw: bool = ..., vars: Optional[_section] = ...) -> List[Tuple[str, str]]: ...
 
-    def set(self, section: str, option: str, value: str) -> None: ...
+    def set(self, section: str, option: str, value: Optional[str] = ...) -> None: ...
 
-    def write(self,
-              fileobject: IO[str],
-              space_around_delimiters: bool = ...) -> None: ...
+    def write(self, fp: IO[str], space_around_delimiters: bool = ...) -> None: ...
 
     def remove_option(self, section: str, option: str) -> bool: ...
 
     def remove_section(self, section: str) -> bool: ...
 
-    def optionxform(self, option: str) -> str: ...
+    def optionxform(self, optionstr: str) -> str: ...
 
 
 class ConfigParser(RawConfigParser):
     def __init__(self,
```

Comment:
As far as I can see in Python 3.7 and 3.8, `ConfigParser` just inherits its `__init__` from `RawConfigParser`. We should do the same in the stubs.